WordPress/src/main/java/org/wordpress/android/ui/mlp/ModalLayoutPickerFragment.kt

```diff
@@ -81,10 +81,13 @@ class ModalLayoutPickerFragment : BottomSheetDialogFragment() {
         }
 
         createBlankPageButton.setOnClickListener {
+            viewModel.createPage()
```

Comment:
NitPick: We might want to consider invoking "viewModel.onCreatePageClicked" instead of "createPage" and "dismiss". The view layer doesn't need to know what happens when the user clicks on a button - it just needs to inform the VM. Wdyt?

Reply:
Thanks for catching this :)
Fixed with 2ac8431075e5f95b46ab9beb0040eac9c4ca09f5